app/backend/src/clj/teet/integration/integration_email.clj

```diff
@@ -93,8 +92,15 @@
 (defn send-email! [msg]
   (let [config (environment/config-map
                 {:subject-prefix [:email :subject-prefix]
-                 :contact-address [:email :contact-address]})]
-    (send-email!*
+                 :contact-address [:email :contact-address]})
+        smtp-node-config (environment/config-map
```

Comment:
you could just use `(environment/config-value :email :server)` to get the whole map, no need to separately get each key. That also makes it more future proof as any possible new keys in the server config don't need to be listed here.